Vagrantfile

```diff
@@ -207,14 +206,43 @@ Vagrant.configure("2") do |config|
     },
     ubuntu: {
       name: 'ubuntu/bionic64',
-      version: '20190514.0.0'
+      version: '20190514.0.0',
+      scripts: [
+        {
+          name: "install-python",
+          type: "shell",
+          data: INSTALL_PYTHON
+        }
+      ]
     }
   }
 
-  INSTALL_PYTHON = <<-SCRIPT
-  #!/bin/bash
-  apt install python -y
-  SCRIPT
+  config.vm.box = os_data[:centos][:name]
+  config.vm.box_version = os_data[:centos][:version]
+
+  config.vm.provider "virtualbox" do |v|
+    v.linked_clone = true
+    v.memory = 2048
+    v.cpus = 2
+    v.customize ["modifyvm", :id, "--chipset", "ich9"]
+  end
+
+  config.vm.network "private_network",
+    type: :dhcp,
+    nic_type: 'virtio',
+    **CONTROL_PLANE_NETWORK
+  config.vm.network "private_network",
+    type: :dhcp,
+    nic_type: 'virtio',
+    **WORKLOAD_PLANE_NETWORK
+
+  config.vm.define :bootstrap, primary: true do |machine|
+    declare_bootstrap machine, os_data[:centos]
+  end
+
+  config.vm.define :bootstrap_ubuntu do |machine|
```

Comment:
Won't we need `autostart: false` on this one as well?

Reply:
Oops indeed.